monkey/infection_monkey/telemetry/messengers/telemetry_messenger_wrapper.py

```diff
@@ -0,0 +1,7 @@
+from infection_monkey.telemetry.i_telem import ITelem
+from infection_monkey.telemetry.messengers.i_telemetry_messenger import ITelemetryMessenger
+
+
+class TelemetryMessengerWrapper(ITelemetryMessenger):
```

Comment:
What is this class responsible for?

Reply:
See https://github.com/guardicore/monkey/pull/1259/files#r658679589

In an upcoming PR, a more complicated messenger will be introduced to allow batching the telemetries together to minimize the load on the network.